### PR TITLE
Final Merge

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates
- [ ] Use MARVEL SUPERHEROES API
- [ ] Follow Nelson's Sakwa design idea shown on Behance
- [ ] Implemented Home Page
- [ ] Implemented Details Page
- [ ] The data retrieved from the API should be stored in the Redux store.
- [ ] Implemented testing using Jest
- [ ] Implemented Snapshot testing
- [ ] Implemented App test
- [ ] Implemented Home test
- [ ] Implemented Navbar test
- [ ] Implemented Search test

image
Implemented Responsive Design 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,44 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This Project is based on Nelson Sakwa design project which you can watch on this [link](https://www.behance.net/sakwadesignstudio)
```
Comment (rustam-tolipov) [inline README.md:43]: [OPTIONAL] Kindly add MIT license to your project.

File: src/Redux/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+// import logger from 'redux-logger';
```
Comment (rustam-tolipov) [inline src/Redux/configureStore.js]: - [ ] To make your code clean. Kindly remove unused commented codes from your file.

File: src/Redux/configureStore.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import herosSlice from './HomeReducer/HomeReducer';
+
+const store = configureStore({
+  reducer: {
+    heros: herosSlice,
+  },
+  // middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
```
Comment (rustam-tolipov) [inline src/Redux/configureStore.js]: Kindly remove unused commented codes.

File: src/__tests__/App.test.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
```
Comment (rustam-tolipov) [inline src/__tests__/App.test.js:1]: I can see all your snapshot tests are passing. 👏👏👏
Screen Shot 2022-10-05 at 09 20 44

However, it's not as expected unless fixing up some shown errors in your console. You can take a look at these examples:

`App.test.js`

Screen Shot 2022-10-05 at 09 20 37

File: src/__tests__/Home.test.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
```
Comment (rustam-tolipov) [inline src/__tests__/Home.test.js:1]: `Home.test.js` 

Screen Shot 2022-10-05 at 09 20 25

File: src/__tests__/Navbar.test.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
```
Comment (rustam-tolipov) [inline src/__tests__/Navbar.test.js:1]: `Navbar.test.js`
`Search.test.js`

Screen Shot 2022-10-05 at 09 20 03

File: src/__tests__/Search.test.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
```
Comment (rustam-tolipov) [inline src/__tests__/Search.test.js:1]: - [ ] One more requirement. It's great you added tests. But it's only snapshot tests. However, in the requirement, you also need to have integration tests as well. 
Highlights: 

- [ ] [1-highlight](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Write%20integration%20tests%20with%20a%20React%20testing%20library.)
- [ ] [2-highlight](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests,the%20Redux%20Store.)
- [ ] [3-highlight](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20the%20tests%20for%20the%20application.)

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @lucabenaventew, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great Design_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rustam-tolipov** [comment]
Kindly reach out via Slack If I missed something important. `Rustamjon Tolipov`

**Comment by lucabenaventew** [comment]
# Project Updates :green_circle: 
image
Tests run properly, tests are implemented in integration, and warnings appear due to npm deprecated packages, kindly accept the project because all the requirements have been completed.
Implemented clean code (Erase comments)
Add MIT License
Improve README

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Luca, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**